sklearn/decomposition/tests/test_pca.py

```diff
@@ -95,24 +96,30 @@ def test_whitening(solver, copy):
 
     X_ = X.copy()
     pca = PCA(
-        n_components=n_components, whiten=False, copy=copy, svd_solver=solver
+        n_components=n_components,
+        whiten=False,
+        copy=copy,
+        svd_solver=solver,
+        random_state=global_random_seed,
     ).fit(X_.copy())
     X_unwhitened = pca.transform(X_)
     assert X_unwhitened.shape == (n_samples, n_components)
 
     # in that case the output components still have varying variances
-    assert X_unwhitened.std(axis=0).std() == pytest.approx(74.1, rel=1e-1)
+    assert X_unwhitened.std(axis=0).std() > 69
```

Comment:
Again, previous comparison probably tailored to seed 0 dataset. I lowered the threshold a bit to pass for all seeds in 0-99 range. Also decided to do greater-than instead of approx equality, as the intent is to confirm varying variances.